### PR TITLE
[SIG-3414] Add Zoom message

### PR DESCRIPTION
**Pull request by aditudorache**
This PR adds a zoom message when the containers are not visible due to the current zoom level.

### CODE REVIEW

File: src/signals/incident/components/form/ContainerSelect/Selector/ZoomMessage/ZoomMessage.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  @media screen and ${breakpoint('max-width', 'tabletM')} {
+    left: 0;
+    width: 100%;
+    margin: 0;
+  }
+`;
+
+export interface ZoomMessageProps {
+  showZoomMessage: boolean;
+  setShowZoomMessage: (show: boolean) => void;
```
Comment (MauritsR) [inline src/signals/incident/components/form/ContainerSelect/Selector/ZoomMessage/ZoomMessage.tsx:29]: I'd simplify the props (e.g. only a maxZoomLevel prop). The component can then handle it's own state. 

Better yet, all logic in this component can be extracted to a hook (e.g. useIsLayerWithinZoomBoundaries)

**Review by MauritsR — CHANGES_REQUESTED** [review]

File: src/signals/incident/components/form/ContainerSelect/Selector/Selector.tsx [hunk 2]
```diff
@@ -89,6 +96,28 @@ const EmptySelectionWrapper = styled.div`
   margin: ${themeSpacing(4)} 0;
 `;
 
+const ButtonBarStyle = styled.div<{ layerVisible: boolean }>`
+  @media screen and ${breakpoint('max-width', 'tabletM')} {
```
Comment (MauritsR) [inline src/signals/incident/components/form/ContainerSelect/Selector/Selector.tsx:100]: This should be 

`  @media screen and (${breakpoint('min-width', 'tabletM')}) {`

Or it won't be triggerred on Mac (also, I get a tslint error for it: `( expected ts-styled-plugin(9999)`)

**Review by MauritsR — CHANGES_REQUESTED** [review]
Looks good! Left one last remark.

File: src/signals/incident/components/form/ContainerSelect/Selector/Selector.tsx (hunk 2 above)
Comment (aditudorache) [reply src/signals/incident/components/form/ContainerSelect/Selector/Selector.tsx]: Good catch! Added brackets! I get that the ts error as well. Not found yet a way to suppress it. It looks that with brackets is gone :-)

**Pull request merged**